### PR TITLE
Make create table test rely on wider supported type

### PR DESCRIPTION
**Pull request by zaz968m**

### CODE REVIEW

**Review by kokosing — APPROVED** [review]
Ping me when automation passes so I can merge it.

File: presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java [hunk 1]
```diff
@@ -204,7 +204,7 @@ public void testCreateTable()
         assertTableColumnNames(tableName, "a", "b", "c");
 
         String tableNameLike = "test_create_like_" + randomTableSuffix();
-        assertUpdate("CREATE TABLE " + tableNameLike + " (LIKE " + tableName + ", d boolean, e varchar)");
```
Comment (findepi) [inline presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java:207]: please add boolean to `io.prestosql.testing.AbstractTestDistributedQueries#testDataMappingSmokeTestData` (can be a follow up)

Comment (zaz968m) [reply presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java:207]: https://github.com/prestosql/presto/pull/5900

**Comment by zaz968m** [comment]
@kokosing cancelled checks are not related to code.

**Comment by kokosing** [comment]
Automation failed with: https://github.com/prestosql/presto/issues/5422

**Pull request merged**